packages/actions/test/e2e/01-setup.test.ts

```diff
@@ -1,58 +1,225 @@
+
+    // Get configs for storage.
+    const { ceremonyBucketPostfix } = getStorageConfiguration()
+    const ceremonyData = fakeCeremoniesData.fakeCeremonyScheduledFixed
+    const bucketName = getBucketName(ceremonyData.data.prefix, ceremonyBucketPostfix)
+    // file to upload
+    const localPath = "/tmp/circuitMetadata.log"
+    fs.writeFileSync(localPath, "test data")
+    const objectName = "zkey.zkey"
+    const duplicateBucketName = randomBytes(10).toString("hex")
+
+    beforeAll(async () => {
+        // create a new user without contributor privileges
+        await createNewFirebaseUserWithEmailAndPw(userApp, user.data.email, userPwd)
+        await sleep(5000)
+
+        // Retrieve the current auth user in Firebase.
+        const currentAuthenticatedUser = getCurrentFirebaseAuthUser(userApp)
+        user.uid = currentAuthenticatedUser.uid
+
+        // create account for coordinator
+        await createNewFirebaseUserWithEmailAndPw(userApp, coordinatorEmail, coordinatorPwd)
+        await sleep(5000)
+
+        const currentAuthenticatedCoordinator = getCurrentFirebaseAuthUser(userApp)
+        coordinatorUid = currentAuthenticatedCoordinator.uid
+
+        // add custom claims for coordinator
+        await setCustomClaims(adminAuth, coordinatorUid, { coordinator: true })
+    })
+
+    it("should fail to create a sample ceremony without being a coordinator", async () => {
+        await signInWithEmailAndPassword(userAuth, user.data.email, userPwd)
+        assert.isRejected(createS3Bucket(userFunctions, bucketName))
+    })
+
+    // run these tests only in production mode
+    if (envType === TestingEnvironment.PRODUCTION) {
+        it("should revert when trying to create a ceremony with an existing prefix", async () => {
+            // login with coordinator creds
+            await signInWithEmailAndPassword(userAuth, coordinatorEmail, coordinatorPwd)
+            const currentAuthenticatedCoordinator = getCurrentFirebaseAuthUser(userApp)
+            // refresh token
+            await currentAuthenticatedCoordinator.getIdToken(true)
+            // Create once
+            await createS3Bucket(userFunctions, duplicateBucketName)
+            await sleep(5000)
+            // Create again
+            expect(createS3Bucket(userFunctions, duplicateBucketName)).to.be.rejectedWith("Failed request.")
+        })
+
+        it("should do a full multi part upload", async () => {
+            // make sure we are logged in as coordinator
+            await signInWithEmailAndPassword(userAuth, coordinatorEmail, coordinatorPwd)
+            // 1. create bucket
+            await createS3Bucket(userFunctions, bucketName)
+
+            // 2. multi part upload
+            const success = await multiPartUpload(
+                userFunctions,
+                bucketName,
+                objectName,
+                localPath,
+                process.env.CONFIG_STREAM_CHUNK_SIZE_IN_MB || "128",
```

Comment:
Avoid using defaults when testing to be sure what values are we using as input for methods.

Reply:
The same is in the `getStorageConfiguration` test utility that was introduced during the test bootstrapping. Actually forgot it existed (and will use now), but that function uses the same default values if the env var is not set

Reply:
Since we have included an initial check on each individual environment variable, it no longer makes sense to have these defaults.